dashboard/src/features/tutorial/reducers.js

```diff
@@ -1,7 +1,11 @@
 import { combineReducers } from 'redux'
+import steps from './steps.json'
 
 export const step = (state = 0, action) => {
-  if (action.type == 'TUTORIAL_NEXT_STEP' || action.type == 'UPDATE_TUTORIAL'){
+  if (action.type == 'TUTORIAL_NEXT_STEP'){
+    return state + 1
+  }
+  else if (action.type == 'UPDATE_TUTORIAL' && steps[state].objectType == action.object) {
```

Comment:
The `else` here is technically unnecessary. If we still want to have the `else`, though, it should appear on the line above, next to the previous closing brace.